### PR TITLE
github: workflows: Turn nightly merge back on

### PR DESCRIPTION
**Pull request by DRuffer-tmo**
Reather than reverting, this will preserve the record of when this was turned off and on.

### CODE REVIEW

**Review by johnlange2 — DISMISSED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -5,6 +5,6 @@
 # Order is important; for each modified file, the last matching
 # pattern takes the most precedence.
 
-/*                                         @johnlange2 @ryanhagen-tmo @eniolaodufuwa-tmo @jtbaumann @DRuffer-tmo @joebairdtmo @atrowbr2-tmo @wtucker6 @JLoucks4 @s-karstein @jvhutch @YHusain1
+/*                                         @johnlange2 @ryanhagen-tmo @eniolaodufuwa-tmo @jtbaumann @DRuffer-tmo @joebairdtmo @atrowbr2-tmo @wtucker6 @JLoucks4 @s-karstein @jvhutch @YHusain1 @rtalbott-tmo amovitz-tmobile rgrundy1
```
Comment (ryanhagen-tmo) [inline CODEOWNERS:8]: Code owners contains errors

Comment (ryanhagen-tmo) [reply CODEOWNERS:8]: add @ to the beginning of each username

Comment (DRuffer-tmo) [reply CODEOWNERS]: I fixed that, but I always see that error on this CODEOWNERS

Comment (DRuffer-tmo) [reply CODEOWNERS]: And I'm not sure "Undefined Kconfig symbols" can be fixed without catching up to the nightly merges 1st. Can we just merge this so I can move on and fix everything that way it should be?

Comment (DRuffer-tmo) [reply CODEOWNERS]: For some reason, I had to add the .github/workflows/tmo-nightly-merge-main-tmo-main.yml file back, but CODEOWNERS got resolved in the Catchup PR that was just merged. so This conversation is resolved.

**Comment by DRuffer-tmo** [comment]
I pull this back into a draft because it's not worth doing until after tomorrow when 3.4 is released and then, I will be using the 3.4 tag rather that the nightly HEAD. After we are happy with that merge, nightly merges can be turned back on for our upstream work.

**Review by eniolaodufuwa-tmo — APPROVED** [review]

**Review by ryanhagen-tmo — APPROVED** [review]

**Pull request merged**